src/cunumeric/sort/sort.cu

```diff
@@ -34,6 +34,20 @@ namespace cunumeric {
 
 using namespace Legion;
 
+class ThrustAllocator : public ScopedAllocator {
+ public:
+  using value_type = char;
+
+  ThrustAllocator(Legion::Memory::Kind kind, bool scoped = true, size_t alignment = 16)
```

Comment:
Can you think of a reason that we would deviate from the default values for `scoped` and `alignment`? Otherwise, let's hide them from the users. If we have an actual use case for either of those, we can always add them back.